features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx

```diff
@@ -229,6 +231,15 @@ const RouteMenuMapping = (intl) => [
             },
         ],
     },
+    {
+        id: intl.formatMessage({
+            id: 'Base.RouteMenuMapping.manage.alerts',
+            defaultMessage: 'Manage Alerts',
+        }),
+        path: '/analytics/manage-alerts',
+        component: () => <ManageAlerts />,
```

Comment:
why we need to use lambda here ? component accept just component class/function, they have a separate prop for render function

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8662/commits/cb1fd45e6fbcb1c4166f90d86ef8f0c9366450f7